eo-maven-plugin/src/main/resources/org/eolang/maven/pre/data.xsl

```diff
@@ -66,6 +66,11 @@ SOFTWARE.
           </xsl:attribute>
           <xsl:choose>
             <xsl:when test="@data='bytes'">
+              <xsl:choose>
+                <xsl:when test="@base='string'">
+                  <xsl:text>new String(</xsl:text>
+                </xsl:when>
+              </xsl:choose>
               <xsl:text>new byte[] {</xsl:text>
```

Comment:
@andreoss I think it's better to put the `new bytes[]...` into a XSL variable and then deal with it. What you did now is not very extendable, since there will be more cases like `string` -- how will the code look in the future?